### PR TITLE
Adding Yazi

### PR DESCRIPTION
**Pull request by hoxbro**
Resolves https://github.com/conda-forge/staged-recipes/issues/28109

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by hoxbro** [comment]
@conda-forge/rust, ready for review!

This is my first time creating a rust conda package, so I'm unsure about the static libraries tasks.

**Comment by hoxbro** [comment]
@conda-forge/help-rust, ready for review. Thank you!

This is my first time creating a rust conda package, so I'm unsure about the static libraries tasks.

File: recipes/yazi/build.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
```
Comment (xhochy) [inline recipes/yazi/build.sh:4]:
```suggestion
export CARGO_PROFILE_RELEASE_STRIP=symbols
export CARGO_PROFILE_RELEASE_LTO=fat

```

File: recipes/yazi/build.sh [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+cargo install --locked --root "${PREFIX}" yazi-fm yazi-cli
```
Comment (xhochy) [inline recipes/yazi/build.sh:5]:
```suggestion
cargo install --no-track --locked --root "${PREFIX}" yazi-fm yazi-cli
```

File: recipes/yazi/build.sh [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+cargo-bundle-licenses --format yaml --output "${SRC_DIR}/THIRDPARTY.yml"
+
+rm "$PREFIX/.crates2.json"
+rm "$PREFIX/.crates.toml"
```
Comment (xhochy) [inline recipes/yazi/build.sh:9]:
```suggestion
```

File: recipes/yazi/bld.bat [hunk 4]
```diff
@@ -0,0 +1,9 @@
+set CARGO_HOME=D:\cargo
```
Comment (xhochy) [inline recipes/yazi/bld.bat:1]:
```suggestion
set CARGO_PROFILE_RELEASE_STRIP=symbols
set CARGO_PROFILE_RELEASE_LTO=fat

```

File: recipes/yazi/bld.bat [hunk 5]
```diff
@@ -0,0 +1,9 @@
...
+cargo install --locked --bins --root %PREFIX% yazi-fm yazi-cli
```
Comment (xhochy) [inline recipes/yazi/bld.bat:2]:
```suggestion
cargo install --no-track --locked --bins --root %LIBRARY_PREFIX% yazi-fm yazi-cli
```

File: recipes/yazi/bld.bat [hunk 6]
```diff
@@ -0,0 +1,9 @@
...
+if %errorlevel% NEQ 0 exit /b %errorlevel%
+cargo-bundle-licenses --format yaml --output %SRC_DIR%/THIRDPARTY.yml
+if %errorlevel% NEQ 0 exit /b %errorlevel%
+del %PREFIX%\.crates2.json
+if %errorlevel% NEQ 0 exit /b %errorlevel%
+del %PREFIX%\.crates.toml
+if %errorlevel% NEQ 0 exit /b %errorlevel%
```
Comment (xhochy) [inline recipes/yazi/bld.bat:9]:
```suggestion
```

**Review by xhochy — CHANGES_REQUESTED** [review]

**Comment by hoxbro** [comment]
@conda-forge/help-rust, ready for review. Thank you!

**Review by xhochy — APPROVED** [review]

**Pull request merged**